introduction/java_loops_i/Solution.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class Solution {
+  static private final int MAX_NUMBER_OPERATIONS = 10;
+  public static void main(String[] args) {
+    Scanner input = new Scanner(System.in);
+    int inputNumber = input.nextInt();
```

Comment:
input number doesn't provide any new information - it's input since you scan it, and number since it's int.